jakarta.ls/src/main/resources/transactions.json

```diff
@@ -1,3 +1,14 @@
 {
-
-}
\ No newline at end of file
+    "@Transactional": {
+      "prefix": "@Transactional()",
```

Comment:
Usually the prefix is a shorter word for ease of use. Also, if you want to introduce other Transactional snippets in the future it could get confusing if the main word is already taken. Persistence uses "persistunsync" and "persistentity", perhaps something analogous would be better.